.github/workflows/test-cron.yaml

```diff
@@ -132,7 +134,9 @@ jobs:
       uses: actions/upload-artifact@v2
       with:
         name: native_engine.so.${{ matrix.python-version }}.${{ runner.os }}
```

Comment:
The name is no longer entirely accurate. Maybe should be `native_code.${...}` (if you change the name on upload you have to change it correspondingly on download).